pkg/diagnostics/network/objects.go

```diff
@@ -64,7 +66,7 @@ func GetNetworkDiagnosticsPod(command []string, podName, nodeName string) *kapi.
 							ReadOnly:  true,
 						},
 					},
-					Command: command,
+					Command: []string{"sh", "-c", cmd},
```

Comment:
`cmd=$2` is not sufficient, we may have more than 2 args. ${@:2} did not work reliably, so switched to 'shift'.